app/src/main/java/com/example/calculator/MyFragment.kt

```diff
@@ -0,0 +1,169 @@
+package com.example.calculator
+
+import android.annotation.SuppressLint
+import android.os.Bundle
+import android.text.SpannableStringBuilder
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.EditText
+import android.widget.Toast
+import androidx.fragment.app.Fragment
+import com.example.calculator.databinding.FragmentMainBinding
+import com.example.myapplication.Parser
+import java.math.BigDecimal
+
+class MyFragment : Fragment() {
+    private var _binding: FragmentMainBinding? = null
+    private val binding
+        get() = requireNotNull(_binding)
+
+
+    private lateinit var display: EditText
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View {
+        _binding = FragmentMainBinding.inflate(inflater, container, false)
+        val view = binding.root
+        return view
+    }
+
+    @SuppressLint("SetTextI18n")
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        display.id = R.id.input
+        display.showSoftInputOnFocus = false
+        /*display.setOnClickListener(View.OnClickListener {
+            if (getString(R.string.display).equals(display.text)) {
+                display.setText("")
+            }
+        })*/
+
+        var bracketsDifferance = 0
+        var bracketsCorrectness: Boolean = true
+
+        with(binding) {
+            val clickListener = View.OnClickListener { p0 ->
```

Comment:
параметры нужно называть корректно
```suggestion
            val clickListener = View.OnClickListener { view ->
```